pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -491,6 +491,13 @@ in
         '';
   };
 
+  vpim = attrs: {
+    dontBuild = false;
+    postPatch = ''
+      sed 1i'# encoding: ISO-8859-1' -i lib/vpim/vcard.rb
+    '';
+  };
```

Comment:
can you introduce this in another commit or PR? My future self would be confused in finding an addition like that in a deletion commit